esp-hal/src/dma/pdma/crypto.rs

```diff
@@ -0,0 +1,525 @@
+use esp32s2::crypto_dma::aes_sha_select::SELECT;
```

Comment:
I don't care personally but just to confirm, was this import intentional? I don't typically see the PAC directly imported like this.